consensus/istanbul/backend/handler.go

```diff
@@ -48,10 +52,14 @@ const (
 	istanbulGetAnnouncesMsg        = 0x16
 	istanbulGetAnnounceVersionsMsg = 0x17
 	istanbulAnnounceVersionsMsg    = 0x18
+	istanbulDirectAnnounceMsg      = 0x19
+	istanbulValidatorProofMsg      = 0x1a
```

Comment:
Could be "SignedVersionedEnode"

Reply:
Do you mean that "VersionedEnodeCertificate" should or shouldn't ever be unsigned?

What do you think of:

`type versionedEnode struct` -> `type versionedEnodeCertificate struct`
`istanbulVersionedEnodeMsg` -> `istanbulVersionedEnodeCertificateMsg`
`istanbulValidatorProofMsg ` -> `istanbulValidatorHandshakeMsg`

Reply:
+1 to that suggestion, although I was going to suggest just `enodeCertificate` rather than `versionedEnodeCertificate`. (The version is a case of a common security measure found in certificates)